mixins/labelled-mixin.js

```diff
@@ -43,7 +43,6 @@ export const LabelMixin = superclass => class extends superclass {
 	disconnectedCallback() {
 		super.disconnectedCallback();
 		this.removeEventListener('d2l-label-change', this._handleLabelChange);
-		if (this._labelObserver) this._labelObserver.disconnect();
```

Comment:
Why is this not needed?  I think I recently added it when I was adding it to https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-observer-mixin.js#L44, which is a similar setup

Reply:
Sorry I meant to call this out -- it wasn't needed because `LabelMixin` never creates an observer, only `LabelledMixin` does.

Reply:
Ah!  Took me way to long to see that, the way GitHub minimizes the diff - and I obviously missed it the first time when adding it.  Should it be moved down to `LabelledMixin`?

Reply:
@dbatiste and I talked a bit about that. The problem with disconnecting it is if it gets reconnected later (maybe via a drag & drop) then it won't start observing again. And that change was larger than I wanted to make here where I just happened to notice it.